airflow/contrib/operators/gcp_container_operator.py

```diff
@@ -170,3 +175,80 @@ def execute(self, context):
         hook = GKEClusterHook(self.project_id, self.location)
         create_op = hook.create_cluster(cluster=self.body)
         return create_op
+
+
+KUBE_CONFIG_ENV_VAR = "KUBECONFIG"
+
+
+class GKEPodOperator(KubernetesPodOperator):
+    template_fields = ('project_id', 'location',
+                       'cluster_name') + KubernetesPodOperator.template_fields
+
+    @apply_defaults
+    def __init__(self,
+                 project_id,
+                 location,
+                 cluster_name,
```

Comment:
Is it possible to pass in a connection id so one can launch a k8s pod in any GKE cluster?

Reply:
I believe what you want can be accomplished by providing the project_id, location, and cluster_name of the cluster that you want to launch the pod in.

I checked and since the base KubernetesPodOperator doesn't use a hook, they have no need for a conn_id, therefore I don't know of a way to specify it.

Reply:
I meant GCP connection, which stores service account key information. The default credential may not work for a (project, zone, cluster-name) tuple, think of starting a k8s pod in some GKE cluster in a different project.